### PR TITLE
Styles and ux fixes

### PR DESCRIPTION
**Pull request by ka1130**
This PR adds some UX & stylistic fine-tuning including:
- adjusting the app for mobiles
- adding info about the channel to the modal and the show's page
- changing the placeholder image for the missing covers with a nicer one
 - custom-styling the `<select>`
 - displaying no pagination if there are less than 12 episodes 
 - redirect to page 1 when user changes the country (so that user doesn't end with empty search results if there are some but not enough to reach a given page)

Link to screencast: https://drive.google.com/file/d/1zz7_9NlZeyp_9GSHMgdgBTw3kFZe1Fva/view

### CODE REVIEW

File: src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss [hunk 1]
```diff
@@ -86,3 +89,9 @@
   text-decoration: underline;
   margin-bottom: 1rem;
 }
+
+@media screen and (min-width: 772px) {
```
[the remark this answers is not in the document]
Comment (ka1130) [reply src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss]: Corrected. :)

File: src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss [hunk 2]
```diff
@@ -23,6 +23,7 @@
       border-radius: 0.5rem;
       border: 0.25rem solid #fff;
       box-shadow: 1px 1px 4px 0px rgba(0,0,0,0.2);
```
[the remark this answers is not in the document]
Comment (ka1130) [reply src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss]: Right, corrected. :)

**Comment by ka1130** [comment]
Thanks, @yakato. :) I've introduced rems and variables for media queries.

File: src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss [hunk 3]
```diff
@@ -22,7 +22,7 @@
       margin-bottom: 2rem;
       border-radius: 0.5rem;
       border: 0.25rem solid #fff;
-      box-shadow: 1px 1px 4px 0px rgba(0,0,0,0.2);
+      box-shadow: 0.125rem 0.125rem 0.25rem 0 rgba(0,0,0,0.2);
```
Comment (yakato) [inline src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss]: 1px is 0.0625rem

Comment (ka1130) [reply src/components/App/ActiveEpisodeModal/ActiveEpisodeModal.module.scss]: Right, even ma calculator sucks on maths :laughing: Correcting. :)

**Pull request merged**